### DISCUSSION TITLE
功能建议

### DISCUSSION THREAD

**Discussion by crazyvest** [Blinue/Magpie#813] [Ideas]
十分感谢MAGPIE开发者的贡献！
利用缩放功能随之的一些渲染特性可以改善一些游戏的性能和兼容性问题，希望此应用能加入在命令行下指定配置文件启动目标游戏的功能，或者自动检测游戏进程启动后自动缩放的功能，这对于快捷启动一些游戏和一些不限制外挂插件的游戏调试十分有用。

**Comment by Blinue** [comment]
命令行功能在 #686 中讨论。

> 自动检测游戏进程启动后自动缩放的功能

为游戏创建配置文件，然后开启位于前台时自动缩放即可。

**Reply by ws2bhhh** [reply]
> 为游戏创建配置文件，然后开启位于前台时自动缩放即可。

不太可靠，启动后无反应需要切换一下窗口才能反应过来。支持类似 ”magpie *配置方案*  *游戏进程*“ 就好了。目前命令行支持进度如何了呢🤔
而且前台的定义是窗口焦点所在吗，我还以为游戏不在最小化状态或者后台就算前台…
